ops-bedrock/docker-compose.yml

```diff
@@ -157,3 +171,16 @@ services:
 #      - "9090:8080"
 #    volumes:
 #      - op_log:/op_log:ro
+
+  #in no way should this be running in production
+  kms:
+    image: bobanetwork/local-kms:latest
```

Comment:
Is this something we rolled ourselves? It looks like it's probably derived from https://github.com/nsmithuk/local-kms -- any reason we're not just referencing the upstream image? What did we change?

Reply:
The image is built by @InoMurko We use this image in our boba repo too. I don't know what changes we made with this one.

Reply:
hey, @jyellick this is the reason :-) https://github.com/nsmithuk/local-kms/pull/38